app/assets/javascripts/archive-uploader.js

```diff
@@ -0,0 +1,30 @@
+const profileFileButton = document.getElementById("profile-file-btn");
+
+const profileFileChosen = document.getElementById("profile-file-chosen");
+
+profileFileButton.addEventListener("change", function() {
+  profileFileChosen.textContent = this.files[0].name;
+  checkProfileUploadButton();
+});
+
+const photosFileButton = document.getElementById("photos-file-btn");
+
+const photosFileChosen = document.getElementById("photos-file-chosen");
+
+photosFileButton.addEventListener("change", function() {
+  photosFileChosen.textContent = this.files[0].name;
+  checkProfileUploadButton();
+});
+
+const checkProfileUploadButton = function() {
+  let photo_files = $("#profile-file-btn")[0].files;
+  let profile_files = $("#photos-file-btn")[0].files;
+
+  if ((photo_files.size + profile_files.size) === 0)
```

Comment:
- Opening curly brace does not appear on the same line as controlling statement.
- Identifier 'photo_files' is not in camel case.
- Identifier 'profile_files' is not in camel case.